purchase_work_acceptance/models/purchase.py

```diff
@@ -12,8 +12,9 @@ class PurchaseOrder(models.Model):
         string='WA count',
         default=0,
     )
-    wa_ids = fields.Many2many(
+    wa_ids = fields.One2many(
         comodel_name='work.acceptance',
+        inverse_name='purchase_id',
         compute='_compute_wa_ids',
         string='Work Acceptances',
```

Comment:
No need of compute here

Reply:
Edited : https://github.com/OCA/purchase-workflow/pull/774/commits/d48b95022bd9ec1b1bfd9832102f755e5cc709d8